### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by puckipedia**
For http://www.google-melange.com/gci/task/view/google/gci2014/5286076935569408

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+objects.*
+/source/lib
```
Comment (waddlesplash) [inline .gitignore:2]: `/source/`?

File: RobinHoudini/RHoudini.rdef [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+	minor  = 0,
+
+	variety = B_APPV_FINAL,
+	internal = 0,
+
+	short_info = "Robin Houdini",
+	long_info = "Robin Houdini"
+};
+
+resource app_flags B_SINGLE_LAUNCH;
```
Comment (waddlesplash) [inline RobinHoudini/RHoudini.rdef]: Please put all of these at the top of the file (and delete file_types, it's useless)

File: .gitignore (hunk 1 above)
Comment (puckipedia) [inline .gitignore:2]: It expected libHTTP there

Comment (puckipedia) [inline .gitignore:2]: And the /server one is where 'make install' installs to

File: source/common/libHTTPHeaders [hunk 3]
```diff
@@ -0,0 +1 @@
+/DataFluffyTail/libHTTP/libHTTPx86/libHTTP/headers/
```
Comment (waddlesplash) [inline source/common/libHTTPHeaders]: Looks like you forgot to remove this. Also, `DataFluffyTail`? LOL

File: .gitignore (hunk 1 above)
Comment (waddlesplash) [inline .gitignore:2]: But why `/`? Can't you remove the `/` then?

Comment (puckipedia) [inline .gitignore:2]: / makes sure it is based in the root of the git repo, just in case

**Pull request merged**